{
  "pygeoogc.exceptions": {
    "__set__": true,
    "elements": [
      "pygeoogc"
    ]
  }
}